### PR TITLE
fix(FEC-11537): Need to re-add fontScale to textStyle config for backward compatability

### PR DESCRIPTION
**Pull request by SivanA-Kaltura**
### Description of the Changes

Re-add fontScale to textStyle for backward compatibility.
Fixes FEC-11537.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: src/track/text-style.js [hunk 1]
```diff
@@ -148,6 +148,7 @@ class TextStyle {
     let textStyle = new TextStyle();
     textStyle.fontEdge = getValue(setting.fontEdge, textStyle.fontEdge);
     textStyle.fontSize = getValue(setting.fontSize, textStyle.fontSize);
+    textStyle.fontScale = getValue(setting.fontScale, textStyle.fontScale);
```
Comment (yairans) [inline src/track/text-style.js:151]: so if app overrides the `fontSize` only, the `fontScale` wiill override it and it won't be changed no?

Comment (SivanA-Kaltura) [reply src/track/text-style.js:151]: In that case, getValue will receive settings.fontScale=undefined, and will fallback to textStyle.fontScale (which was set by fontSize)

Comment (yairans) [reply src/track/text-style.js:151]: great. do we have unit tests for both fontSize and fontScale overrriding?

Comment (SivanA-Kaltura) [reply src/track/text-style.js:151]: Yes, see my changes in player.spec.js

Comment (SivanA-Kaltura) [reply src/track/text-style.js:151]: Yes, see my changes to player.spec.js

**Review by yairans — APPROVED** [review]

**Pull request merged**